dev/service-registry/service-locator/src/main/scala/com/lightbend/lagom/discovery/ServiceLocatorServer.scala

```diff
@@ -75,8 +75,8 @@ class ServiceLocatorServer extends Closeable {
   }
 
   def serviceLocatorAddress: URI = {
-    // Converting InetSocketAddress into URL is not that simple. 
-    // Because we know the service locator is running locally, I'm hardcoding the hostname and protocol. 
+    // Converting InetSocketAddress into URL is not that simple.
+    // Because we know the service locator is running locally, I'm hardcoding the hostname and protocol.
     new URI(s"http://localhost:${server.mainAddress.getPort}")
   }
 
```

Comment:
If we make the host configurable, a user may choose their public IP (to run tests from a colleague machine). In that case these methods need to use the configured value:

```scala
  def serviceLocatorAddress: URI = {
    new URI(s"http://localhost:${server.mainAddress.getPort}")
  }
 
  def serviceGatewayAddress: URI = {
    new URI(s"http://localhost:${gatewayAddress.getPort
  }
}
```


This is not a big deal since this methods are only used for logging, but still we could log the truth. :)

Reply:
I noticed this. I am glad you brought it up because the comments made me a bit nervous. :-) I think we can just `s/localhost/${config.host}/` -- do you agree?